e2e-tests/node-client.js

```diff
@@ -94,9 +106,9 @@ class NodeClient {
      */
     async unsafeTransfer(to, amount, nonce, privateKey){
         const from = await Utils.getKeyringPair(privateKey);
-        const U64 = UInt.with(64, "U64");
-        amount = u8aToHex((new U64(TypeRegistry, amount)).toU8a());
-        const signedTx = await Utils.signTransaction(from, to, amount, nonce);
+        const rawNonce = !nonce ? "0x" : u8aToHex((new U64(TypeRegistry, nonce)).toU8a());
+        amount = !amount ? "0x" : u8aToHex((new U64(TypeRegistry, amount)).toU8a());
```

Comment:
Maybe we can use a separate variable for the amount similar to `rawNonce`, f.e having `rawAmount`, just so that we are consistent